src/root.component.tsx

```diff
@@ -26,15 +26,21 @@ function Root(props) {
           showPatientSummary={setShowPatientSummary}
         />
       </Route>
-      <Route
-        path="/patient/:patientUuid/chart"
-        exact
-        component={PatientChartSummary}
-      />
-      <Route
-        path="/patient/:patientUuid/chart"
-        render={routeProps => <LevelTwoRoutes match={props.match} />}
-      />
+      <main className="omrs-main-content">
+        {showPatientSummary && (
+          <Route
+            path="/patient/:patientUuid/chart"
+            exact
+            component={PatientChartSummary}
+          />
+        )}
+        {showPatientSummary && (
+          <Route
+            path="/patient/:patientUuid/chart"
+            render={routeProps => <LevelTwoRoutes match={props.match} />}
```

Comment:
```suggestion
            component={LevelTwoRoutes}
```

Or (equivalently):

```suggestion
            render={routeProps => <LevelTwoRoutes {...routeProps} />}
```

Reply:
Hmm I like the idea of using a spread operator, but that means we will have to change the type of LevelTwoRoutesProps to the type of routeProps, which is okay I guess.